### PR TITLE
Use PackedSequence for ConvRNN

### PR DESCRIPTION
**Pull request by daemon**
Zero-padding is technically incompatible with the recurrent layers, since the hidden outputs would still be altered. Using PackedSequence is the correct way to deal with this issue. Thus, I've invalidated the past results and will report the current ones as soon as the model finishes training.

### CODE REVIEW

File: conv_rnn/train.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 import random
 
 from torch import utils
-from torch.optim.lr_scheduler import ReduceLROnPlateau
+from tqdm import tqdm
```
Comment (tuzhucheng) [inline conv_rnn/train.py]: Be sure to add `tqdm` to `requirements.txt`

**Comment by tuzhucheng** [comment]
By the way please migrate to `SST1` in the common `datasets` module when you get a chance?

**Comment by Victor0118** [comment]
LGTM. Might merge after you push some reasonable results to ensure it works well.

**Comment by daemon** [comment]
@tuzhucheng Sounds good, I'll do that in a future PR.

**Comment by daemon** [comment]
Also, I've added results using the default hyperparameters specified in the paper. They're around 2.7 points off.
